Script/AtomicEditor/ui/frames/inspector/DataBinding.ts

```diff
@@ -479,6 +479,7 @@ class DataBinding {
 
         if (ev.type == Atomic.UI_EVENT_TYPE_CHANGED) {
             if (this.widget == ev.target || this.widget.isAncestorOf(ev.target)) {
+                EditorUI.getMainFrame().resourceframe.currentResourceEditor.setModified(true);
```

Comment:
There is probably a better way of handling this than a really long lookup like this...